bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/di/DataModule.kt

```diff
@@ -0,0 +1,35 @@
+package io.bucketeer.sdk.android.internal.di
+
+import android.app.Application
+import androidx.annotation.VisibleForTesting
+import com.squareup.moshi.Moshi
+import io.bucketeer.sdk.android.internal.model.jsonadapter.EventAdapterFactory
+import io.bucketeer.sdk.android.internal.model.jsonadapter.EventTypeAdapter
+import io.bucketeer.sdk.android.internal.model.jsonadapter.MetricsEventAdapterFactory
+import io.bucketeer.sdk.android.internal.model.jsonadapter.MetricsEventTypeAdapter
+import io.bucketeer.sdk.android.internal.model.jsonadapter.ReasonTypeAdapter
+import io.bucketeer.sdk.android.internal.model.jsonadapter.SourceIDAdapter
+import io.bucketeer.sdk.android.internal.model.jsonadapter.UserEvaluationsStateAdapter
+
+internal class DataModule(
```

Comment:
Same as `:bucketeer` module, this class will provide ApiClient, SQLiteOpenHelper, and SharedPreferences. 

https://github.com/ca-dp/bucketeer-android-sdk/blob/master/bucketeer/src/main/java/jp/bucketeer/sdk/di/DataModule.kt